circe-circumference.js

```diff
@@ -0,0 +1,4 @@
+const circleCircumference = (radius) => {
+  return `${2 * radius * Math.PI}`;
```

Comment:
Here you are unnecessarily converting a number to a string. You could just return the number instead:
```suggestion
  return 2 * radius * Math.PI;
```